src/views/ProductDetail.vue

```diff
@@ -59,102 +59,77 @@
-                  <ion-button @click="getOtherStoresInventoryDetails()" fill="outline">100 ATP</ion-button>
+                  <ion-button @click="getOtherStoresInventoryDetails()" fill="outline">{{ otherStoresInventory }}</ion-button>
                 </ion-item>
                 <ion-item lines="none">
                   <ion-label class="ion-text-wrap">{{ translate("Warehouse")}}</ion-label>
-                  <ion-note slot="end">100 ATP</ion-note>
+                  <ion-note slot="end">{{ warehouseInventory }}</ion-note>
                 </ion-item>
               </ion-list>
             </div>
           </section>
         </div>
-    
-        <div>
-          <h3> {{ translate({  },"order reservtions at the store")}} </h3>
+        
+        <div v-if="orders.length > 0">
+          <h3>{{ translate('order reservations at the store', { count: getInventoryInformation(currentVariant.productId).reservedQuantity ?? '0' }) }}</h3>
           <div class="reservation-section">
+            <div v-for="(order, index) in orders" :key="index">
+              <ion-card> 
+                <ion-item lines="none">
+                  <ion-label class="ion-text-wrap"> {{ order.orderId }}
+                    <p>{{ order.customer ? order.customer.name : '' }}</p>
+                  </ion-label>
+                  <ion-badge color="primary" slot="end">
+                    {{ order.shipmentMethod.shipmentMethodTypeDesc ? order.shipmentMethod.shipmentMethodTypeDesc : order.shipmentMethod.shipmentMethodTypeId }}
+                  </ion-badge> 
+                </ion-item>
 
-            <ion-card>
-              <ion-item lines="none">
-                <ion-label class="ion-text-wrap"> Order ID
-                  <p>Customer name</p>
-                </ion-label>
-                <ion-badge color="primary" slot="end">Open</ion-badge> 
-              </ion-item>
-              <ion-item lines="none">
-                <ion-thumbnail slot="start">
-                  <DxpShopifyImg size="small" />
-                </ion-thumbnail>
-                <ion-label class="ion-text-wrap">
-                  <h4>BRAND</h4>
-                  <h3 class="ion-text-wrap">Virtual name</h3>
-                </ion-label>
-                <ion-note slot="end"> 2 units </ion-note>
-              </ion-item>
-              <ion-list-header color="light">
-                <ion-label>Order items</ion-label>
-              </ion-list-header>
-              <ion-item lines="none">
-                <ion-thumbnail slot="start">
-                  <DxpShopifyImg size="small" />
-                </ion-thumbnail>
-                <ion-label class="ion-text-wrap" >
-                  <p class="overline">BRAND</p>
-                  <h3 class="ion-text-wrap">Virtual name</h3>
-                </ion-label>
-              </ion-item>
-            </ion-card>
+                <ion-item lines="none" v-for="(item, index) in order.currentItem" :key="index"> 
```

Comment:
Instead of looping over the current item sum the total item quantity and display it as a single item.